src/index.ts

```diff
@@ -260,68 +292,56 @@ function transformGlimmer(ast: any, { env }: TransformerContext) {
   let { staticAttrs } = env.customizations
 
   visit(ast, {
-    AttrNode(
-      attr: any,
-      _parent: any,
-      _key: string | null,
-      _index: number,
-      meta: Record<string, any>,
-    ) {
+    AttrNode(attr: any, _path: any, meta: Record<string, any>) {
```

Comment:
In my testing this gets inferred correctly, the `meta` even technically maps to `Partial<...>` but the implementation doesn't care about that. So I think this is a bit nicer:

```suggestion
    AttrNode(attr, _path, meta) {
```